### PR TITLE
feature/SC-3244/calender request only needed time slot

### PR DESCRIPTION
**Pull request by CeEv**
# Checkliste

- [x] https://ticketsystem.schul-cloud.org/browse/SC-3244
- must also merged: https://github.com/schul-cloud/schulcloud-calendar/pull/35 -> done
- [x] WIP PR-Label entfernt, wenn die Checkliste abgearbeitet wurde

todo:
- [ ] test other calendar requests and set time slots

Please not confiused, base branch has other focus. Must is also matched the perfomance ticket.

## Mehr
Weitere Informationen zur DoD [hier im Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

File: helpers/recurringEvents.js [hunk 1]
```diff
@@ -114,23 +113,24 @@ const mapEventProps = (event, req) => {
...
 		});
 	}
 
 	if (event['x-sc-teamId']) {
-		return api(req).get(`/teams/${event['x-sc-teamId']}`).then((team) => {
+		// bad fix for ,<id> error in teams, TODO: fix dashboard
+		let id = event['x-sc-teamId'];
+		if (id.substring(0, 1) === ',') {
+			id = id.substr(1);
+		}
```
Comment (Parnswir) [inline helpers/recurringEvents.js]: Why not fix this where the call  occurs, i.e. in the dashboard?

**Comment by Parnswir** [comment]
I found the issue with commas in the teamId. A fix will be part of the BBB PR #1571

**Comment by CeEv** [comment]
> 
> 
> I found the issue with commas in the teamId. A fix will be part of the BBB PR #1571

Great, i remapped the pr to new ticket. It only contain the time slots by calendar requests and linter.

**Comment by malcm** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 131
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=4850454&bid=16095367)

**Pull request merged**